content/en/docs/Examples/services.md

```diff
@@ -562,6 +669,33 @@ service:
       icon: https://github.com/jgraph/drawio/raw/dev/src/main/webapp/images/drawlogo-color.svg
 ```
 
+## Lidarr/Lidarr
+Source: https://github.com/Lidarr/Lidarr
+
+- deployed_version - Requires an `API_KEY` which can be retrieved at `Settings/General/Security/API Key`
+```yaml
+service:
+  Lidarr:
+    options:
+      semantic_versioning: false
+    latest_version:
+      type: github
+      url: Lidarr/Lidarr
+      url_commands:
+        - type: regex
+          regex: v([0-9.]+)$
+    deployed_version:
+      method: GET
+      url: https://lidarr.example.io/api/v1/system/status
+      headers:
+        - key: X-Api-Key
+          value: <API_KEY>
+      json: version
+    dashboard:
+      icon: https://raw.githubusercontent.com/walkxcode/dashboard-icons/main/png/lidarr.png
```

Comment:
```suggestion
      icon: https://raw.githubusercontent.com/Lidarr/Lidarr/develop/Logo/1024.png
```

Would prefer to keep it all in the same repo. I don't mind if you'd prefer the Lidarr.svg in that dir (some things have trouble with SVG, so I kept it as a PNG like you had)